ibis/udf/vectorized.py

```diff
@@ -104,11 +104,28 @@ def elementwise(input_type, output_type):
     ... def my_string_length(series):
     ...     return series.str.len() * 2
 
-    Define a UDF with non-column parameters:
+    Define an UDF with non-column parameters:
 
     >>> @elementwise(input_type=[dt.string], output_type=dt.int64)
     ... def my_string_length(series, *, times):
     ...     return series.str.len() * times
+
+    Define and use an UDF with multiple return columns:
+
+    >>> @elementwise(
+    ...     input_type=[dt.string],
+    ...     output_type=dt.Struct(['year', 'monthday'], [dt.string, dt.string])
+    ... )
+    ... def year_monthday(date):
+    ...     result = pd.concat(
+    ...         [date.str.slice(0, 4), date.str.slice(4, 8)],
+    ...         axis=1
+    ...     )
+    ...     result.columns = ['year', 'monthday']
```

Comment:
what happens if the result is NOT named, do I get an exception?

Reply:
Actually, results are not required to be named. I updated the test to reflect this. Column labels will be assign based on output_type.